templates/drawers.mustache

```diff
@@ -62,11 +62,11 @@
 
 <body {{{ bodyattributes }}}>
 {{> core/local/toast/wrapper}}
-<div id="page-wrapper" class="d-print-block">
+<div id="page-wrapper" class="d-print-block region-block-elements">
 
     {{{ output.standard_top_of_body_html }}}
 
-    {{> theme_boost/navbar }}
+    {{> theme_boost_union/navbar }}
```

Comment:
Based on the changes to this file, please add 
```
* Include theme_boost_union/navbar template instead of theme_boost/navbar
* Added additional block regions (including JS for offcanvas region)
```
to the file's modification header.